module/src/components/switchInput/switchInput.component.tsx

```diff
@@ -91,9 +117,10 @@ export const SwitchInput = React.forwardRef<HTMLInputElement, ISwitchInputProps>
               {...nativeProps}
               ref={ref}
               type="checkbox"
-              onChange={(event) => setBoundValue(event.currentTarget.checked)}
+              onChange={() => ''}
```

Comment:
Is this legit? Seems odd

Reply:
haaa yeah I mean it is and it isn't... it's to stop React from complaining that onChange isn't set when value/checked is, but it would make more sense to be an empty void function which does mean adding a lint ignore

I have a feeling I did that as a quick way of getting rid of the linting rule to see if what I was doing elsewhere would work and never went back to make it more proper